### ISSUE TITLE
Question: Skipping the Canvas in hover hierarchy

### ISSUE THREAD

**Issue by ankri** [prevwong/craft.js#23]
tl;dr: Example on [CodeSandbox](https://codesandbox.io/s/craft-js-canvas-irgfw)

n my project I make heavy use of the `onRender` callback. For example I add an indicator to every component which shows the name of the component.

This works fine for all Components but Connected Container Components (like `Card` in the Codesandbox example). Those components do not show the Components name `"Card"` but they show `"Canvas"` in the indicator.

I found a workaround by adding `pointer-events: none` to all the `Canvas`es (uncheck the checkbox in the Codesandbox example).

The workaround works and I didn't find any drawbacks, yet. Do you have an idea on how to effectively hover the `UserComponent` and not the `Canvas`?

**Comment by prevwong** [comment]
Oh, this is an expected behaviour since the `Canvas#card-body` inside `Card` is being hovered. You could add some padding/margin. Alternatively, Since the `Card` component seems to only encompass `Canvas#card-body`, what you could do is just:

```jsx
// Card is CardBody previously
const Card = ({ children }) => {
  const {
    connectors: { connect }
  } = useNode();

  return (
    <div ref={connect} className="card-body">
      {children}
    </div>
  );
};

Card.craft = {
  rules: {
    canMoveIn: () => true,
    canDrag: () => false
  }
};

// DroppableCard is Card previously
export const DroppableCard = ({ children }) => {
  return (
      <Canvas is={CardBody} id="card-body">
        {children}
      </Canvas>
  );
};
```

At the moment, the name/displayName of all `<Canvas />` is  "Canvas". But I think it should be set to the element specified in the `is` prop. I will add this enhancement soon, then it should be okay. ✌️

**Comment by ankri** [comment]
Hi,

thank you very much for your feedback. I actually had the use-case to hide the DropTarget (Canvas) in the hierarchy from the user. In my case the `Card` is a container with some other props like e.g. `header`. In this case I wanted to hide from the user that the DropTarget is a component because it is not important and could lead to confusion. 

 But I will re-think my use case and remove or keep my hack.

Feel free to close the issue.

PS: Keep being awesome. I really love your library and your thorough answers 👍
